skaffold-plugins-core/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Interesting. Never seen this. What is this for?

Reply:
This is to make URL (a final class) mockable.

Reply:
Very good to know, thanks!